pkg/controller/installation/typha_autoscaler.go

```diff
@@ -78,25 +76,23 @@ func newTyphaAutoscaler(client client.Client, options ...typhaAutoscalerOption)
 
 // getExpectedReplicas gets the number of replicas expected for a given node number.
 func (t *typhaAutoscaler) getExpectedReplicas(nodes int) int {
+	const maxNodesPerTypha = 200
 	switch {
-	case nodes <= 1:
+	case nodes <= 4:
 		return 1
-	case nodes <= 2:
+	case nodes <= 10:
 		return 2
-	case nodes <= 3:
+	case nodes <= 100:
 		return 3
 	case nodes <= 250:
 		return 4
 	case nodes <= 500:
 		return 5
 	case nodes <= 1000:
 		return 6
-	case nodes <= 1500:
-		return 7
-	case nodes <= 2000:
-		return 8
+	default:
+		return (nodes / maxNodesPerTypha) + 1
```

Comment:
I think we should set a hard cap on the number of typhas that we'll run via this calclulation - say at 20.

Reply:
please add a cap on the number of typhas like Casey suggested.
Also I think this means we'll be at 6 typhas until 1200 since I believe the division is integer math meaning 1199/200 = 5, I think that is ok but I thought I should point that out.